packages/design-system/src/components/Link/Link.module.scss

```diff
@@ -0,0 +1,70 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.link {
+	font: tokens.$coral-paragraph-m;
+	color: tokens.$coral-color-accent-text;
+	background: none;
+	border: none;
+	border-bottom-color: currentColor;
```

Comment:
why the border-bottom-color, but no border ?

Reply:
for when there will be a border (on `:hover`, on `:focus` and on `:active`)